### DISCUSSION TITLE
NEED CAUB LINKS GLITCH.ME links dont work

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#925] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I need CAUB links that are not Glitch or replit smth that doesn't look like a proxy

**Comment by AshtonDavies** [comment, marked as answer]
https://caub.netlify.app
https://caub.vercel.app
https://caub.pages.dev

But I would suggest downloading the page on another device. It doesn't use internet for anything.

**Reply by CGRmags** [reply]
@AshtonDavies wydm downgrading the page? also do you aheva vid of you doing caub bc it keeps saying error or smth

**Reply by AshtonDavies** [reply]
I wrote "downloading," but if you don't know how to do that, try saving with Ctrl+S and then opening from your filesystem.

**Reply by PenguinLord561** [reply]
Make sure to download it as a html file not a mhtml file

**Comment by CGRmags** [comment]
yeah maybe I wont do this i don't get it

On Tue, Feb 13, 2024 at 10:10 AM Ashton Davies ***@***.***>
wrote:

> I wrote "downloading," but if you don't know how to do that, try saving
> with Ctrl+S and then opening from your filesystem.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/925#discussioncomment-8457200>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBFASG47DHXS43WLAB3YTOUAXAVCNFSM6AAAAABDGXKNBOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DINJXGIYDA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.
